### ISSUE TITLE
Interesting comparison between owner classification and owner name search filter

### ISSUE THREAD

**Issue by amabdallah** [WSWCWaterDataExchange/MappingStatesDataToWaDE2.0#156]
Here is an interesting query search as FYI for owner name ="reclam" which returned ~2,350 water rights compared to ~2,100 rights classified as reclamation. 

Two observations about the owner search name filer
1. It works well in getting all owners with "reclam." Getting Lower and Upper case + part of the term.
2. It's not smart compared to the classification code. It does return owners that are not related to the Bureau of Reclamation, such as this owner who happens to have "Reclamation." Pima County Regional Wastewater Reclamation Department
3. The owner classification is useful, and it could be refined over time as we go.

**reclam owner name search**
https://westdaat.westernstateswater.org/?state=N4IgtghgDiBcoC8D2SwBkCmA3DAbOAbAHQCMAHACwDsArAAwECcZJVATGwMwEA0IuEAC4BLQQFcAJhjgVORKgTqcydEiQ5sKNEn1xIAdgHNRk6bAC0akkTJkmBBRxI1tFAL58A7gCc4oAEYY%2BhgAZsIAxsIQuACqAM4YcXAA2gC6fEiewd4AwgJxccJh4ULCBkmwaXzReiUiBgDyWRi%2BsCDeGOECYCBeQi0Aykhi3uEYACoAnlCJKekgcYL9FVXtwjjeAEIQhfoAchBgs5XzUEi4k7jCwXN8%2BrgSwgBiwriCLQAiQhBw%2BmK4uDcbiAA

**Classified as Reclamation**
https://westdaat.westernstateswater.org/?state=N4IgtghgDiBcoC8D2SwBkCmA3DAbOAbAHQCMAHACwDsArAAwECcZJVATGwMwEA0IuEAC4BLQQFcAJhjgVORKgTqcydEiQ5sKNEn1xIAdgHNRk6bAC0akkTJkmBBRxI1tFAL58A7gCc4oAEYY%2BhgAZsIAxsIQuACqAM4YcXAA2gC6fEiewd4AwgJxccJh4ULCBkmwySAAQmLeGBBiAAQAShjhApAiBk0AFDEAytUtAJQg6SCeQhjeA0h14RgAKgCeUIkpE3GC0xVpfN7CON7VEIX6AHIQYBuVE1BIuCu4wsGbfPq4EsIAYsK4ghmABEhBA4PoxLhcG43EA

**Comment by amabdallah** [comment]
Interesting. So "Reclam" only is not specific enough and includes Reclamation unrelated to USBOR. So the owner classification code is working well.


![image](https://github.com/WSWCWaterDataExchange/MappingStatesDataToWaDE2.0/assets/3268971/a26d0008-1bf8-49d5-bdb6-6c9f6f882a85)

**Comment by rwjam** [comment]
Correct.  Those are state native Reclaim departments.  They are not the USBR.

**Comment by amabdallah** [comment]
Let's revisit this. Adel to add details later 

https://www.arcgis.com/apps/mapviewer/index.html?layers=1df76b35789f448094cec79c53c42555

![image](https://github.com/WSWCWaterDataExchange/MappingStatesDataToWaDE2.0/assets/3268971/90302759-c7ca-41c0-a774-a316038e9b8f)
